src/components/VideoPlayerPreview/index.tsx

```diff
@@ -12,30 +13,32 @@ import useWindowDimensions from '@hooks/useWindowDimensions';
 import CONST from '@src/CONST';
 import VideoPlayerThumbnail from './VideoPlayerThumbnail';
 
-const propTypes = {
-    videoUrl: PropTypes.string.isRequired,
+type VideoDimensions = {
+    width: number;
+    height: number;
+};
 
-    videoDimensions: PropTypes.shape({
-        width: PropTypes.number.isRequired,
-        height: PropTypes.number.isRequired,
-    }),
+type VideoPlayerPreviewProps = {
+    /** Url to a video. */
+    videoUrl: string;
 
-    videoDuration: PropTypes.number,
+    /** Dimension of a video. */
+    videoDimensions: VideoDimensions;
 
-    thumbnailUrl: PropTypes.string,
+    /** Duration of a video. */
+    videoDuration: number;
 
-    fileName: PropTypes.string.isRequired,
+    /** Url to a thumbnail image. */
+    thumbnailUrl?: string;
 
-    onShowModalPress: PropTypes.func.isRequired,
-};
+    /** Name of a video file. */
+    fileName: string;
 
-const defaultProps = {
-    videoDimensions: CONST.VIDEO_PLAYER.DEFAULT_VIDEO_DIMENSIONS,
-    thumbnailUrl: undefined,
-    videoDuration: 0,
+    /** Callback executed when modal is pressed. */
+    onShowModalPress: (event?: GestureResponderEvent | KeyboardEvent) => void | Promise<void>;
 };
 
-function VideoPlayerPreview({videoUrl, thumbnailUrl, fileName, videoDimensions, videoDuration, onShowModalPress}) {
+function VideoPlayerPreview({videoUrl, thumbnailUrl, fileName, videoDimensions = CONST.VIDEO_PLAYER.DEFAULT_VIDEO_DIMENSIONS, videoDuration = 0, onShowModalPress}: VideoPlayerPreviewProps) {
```

Comment:
```suggestion
function VideoPlayerPreview({videoUrl, thumbnailUrl, fileName, videoDimensions, videoDuration, onShowModalPress}: VideoPlayerPreviewProps) {
```

Regarding [this discussion](https://github.com/Expensify/App/pull/37425/files#r1526119502): So, if `videoDimensions` and `videoDuration` are always defined in the components that use `VideoPlayerPreview`, we need to remove the default values since they won't make sense anymore.